package.json

```diff
@@ -48,11 +48,10 @@
     ]
   },
   "dependencies": {
-    "@mapbox/mapbox-gl-geocoder": "^4.4.1",
-    "resize-observer-polyfill": "^1.5.1"
```

Comment:
Browser compatibility has improved, we don't need the polyfill https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver#browser_compatibility